parsing lables of override-account/subaccount-id (#1781)
kyma-project/control-plane: docs/kyma-environment-broker/03-06-runtime-overrides.md

## ConfigMap

> The overrides mechanism selects ConfigMaps by filtering the resources using labels. You can prepare overrides for three kinds of combination:

Reviewer: ```suggestion
The overrides mechanism selects ConfigMaps by filtering the resources using labels. You can prepare overrides for three kinds of combinations:
```

> 1. a given plan and Kyma version using the `overrides-plan-{PLAN_NAME}: "true"` and `overrides-version-{KYMA_VERSION}: "true"` labels.

Reviewer: ```suggestion
- A given plan and a Kyma version using the `overrides-plan-{PLAN_NAME}: "true"` and `overrides-version-{KYMA_VERSION}: "true"` labels.
```

Author: modified it.

> 2. a given global account ID and Kyma version using the `overrides-account-{GLOBAL_ACCOUNT_ID}: "true"` and `overrides-version-{KYMA_VERSION}: "true"` labels.

Reviewer: ```suggestion
- A given global account ID and a Kyma version using the `overrides-account-{GLOBAL_ACCOUNT_ID}: "true"` and `overrides-version-{KYMA_VERSION}: "true"` labels.
```

> 3. a given subaccount ID and Kyma version using the  `overrides-account-{SUB_ACCOUNT_ID}: "true"` and `overrides-version-{KYMA_VERSION}: "true"` labels.

Reviewer: ```suggestion
- A given subaccount ID and a Kyma version using the  `overrides-account-{SUB_ACCOUNT_ID}: "true"` and `overrides-version-{KYMA_VERSION}: "true"` labels.
```

Author: changed it.

> **NOTE:** Each ConfigMap that defines overrides must have both labels assigned.

Optionally, you can narrow the scope of the overrides to a specific component. Use the `component: "{COMPONENT_NAME}"` label to indicate the component.

The overrides lookup mechanism requires at least one ConfigMap present for each plan and version pair. Otherwise, it fails.

> See the examples of a ConfigMap：

Reviewer: ```suggestion
See the examples of different ConfigMaps:
```

Author: Fixed.

> 1. Configmap with global overrides for the `trial` plan and versions `1.15.1` and `1.16.0`:

Reviewer: ```suggestion
- ConfigMap with global overrides for the `trial` plan and versions `1.15.1` and `1.16.0`:
```

Author: Done.

  global.disableLegacyConnectivity: "true"
```

> 2. Configmap with global overrides for the `global account` ID and versions `1.15.1` and `1.16.0`:

Reviewer: ```suggestion
- ConfigMap with global overrides for the global account ID and versions `1.15.1` and `1.16.0`:
```

Author: Done.

```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  labels:
    overrides-account-d9994f8f-7e46-42a8-b2c1-1cddd9d2ef06: "true"
    overrides-version-1.15.1: "true"
  name: global-overrides
  namespace: kcp-system
data:
  global.disableLegacyConnectivity: "true"
```

> 2. Configmap with global overrides for the `subaccount` ID and versions `1.16.0`:

Reviewer: ```suggestion
- ConfigMap with global overrides for the subaccount ID and version `1.16.0`:
```

Author: Done.
